### PR TITLE
fixes

### PR DESCRIPTION
**Pull request by anmolbabu**
Signed-off-by: anmolbabu <anmolbudugutta@gmail.com>

### CODE REVIEW

**Comment by anmolbabu** [comment]
@Tendrl/tendrl-core Please review

**Comment by anmolbabu** [comment]
Verified

File: tendrl/performance_monitoring/aggregator/summary.py [hunk 1]
```diff
@@ -135,6 +135,7 @@ def calculate_host_summary(self, node):
             alert_count=0
         )
         try:
+            tendrl_ns.etcd_orm.client.read(old_summary.value)
             old_summary.load()
```
Comment (r0h4n) [inline tendrl/performance_monitoring/aggregator/summary.py]: load returns a refreshed object of the same, if you are not using it why are you loading it?

Comment (anmolbabu) [inline tendrl/performance_monitoring/aggregator/summary.py]: Done

**Pull request merged**